CHANGELOG.md

```diff
@@ -42,6 +45,7 @@ You can find and compare releases at the [GitHub release page](https://github.co
 - Add SDL validation rule `UniqueOperationTypes` (#995)
 - Add ability to remove custom validation rules after adding them via `DocumentValidator::removeRule()`
 - Allow lazy enum values
+- Make `Node` implement `JsonSerializable`
```

Comment:
can we specify why eventually? If I was to chose I'd not add the interface

Reply:
Added a docblock to explain why it is useful